src/conditionals/simple_programs_with_decision_points/hard/q9/GradeToLetter.java

```diff
@@ -0,0 +1,55 @@
+package conditionals.simple_programs_with_decision_points.hard.q9;
+
+import java.util.Scanner;
+
+/*
+ Suppose there is a Java program that takes <tt>grade</tt> as integer input from the user and prints the letter correspondingly 
+ using the if-else statement below. Translate the following statement into a switch statement block, and use the sentence 
+ “Enter the numerical grade: “ to ask for input.
+
+<code>
+        if (grade >= 90) {
```

Comment:
I don't think we should give them this code block because they could just copy paste the code block in and get full marks with no effort.